environment_setup/setup_software.sh

```diff
@@ -109,7 +109,7 @@ fi
 
 rosdep update
 # Install all required dependencies to build this repo
-rosdep install --from-paths $CURR_DIR/../src --ignore-src --rosdistro melodic -y
+rosdep install --from-paths $CURR_DIR/../src --ignore-src --rosdistro melodic -y --os=ubuntu:bionic
```

Comment:
Just a quick comment on the purpose of this, a weeks ago when we had to run `setup_software.sh` again, it was failing for Linux Mint. This quick fix by Henry removed the error